documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -0,0 +1,101 @@
+// This module is included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-deploying-cruise-control-{context}']
+= Deploying Cruise Control
+
+Deploy Cruise Control to your {ProductName} cluster through the `cruiseControl` property in the `Kafka` resource.
+
+Refer to the xref:ref-sample-kafka-resource-config-deployment-configuration-kafka[sample Kafka YAML configuration] for an overview of the `Kafka` resource and its properties.
+
+You can configure `cruiseControl` properties as part of a deployment or redeployment of a Kafka cluster.
+
+.Prerequisites
+
+* A Kubernetes cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `cruiseControl` property of the `Kafka` resource.
++
+The properties you can configure are shown in this example configuration:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  cruiseControl:
+    replicas: 1 <1>
+    capacity: <2>
+      networkIn: 10000
+      networkOut: 10000
+      # ...
+    config: <3>
+      default.goals: >
+         com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,
+         com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal
+      cpu.balance.threshold: 1.1
+      metadata.max.age.ms: 300000
+      send.buffer.bytes: 131072
+      # ...
+    resources: <4>
+      requests:
+        cpu: 200m
+        memory: 64Mi
+      limits:
+        cpu: 500m
+        memory: 128Mi
+    logging: debug <5>
+    template: <6>
+      pod:
+        metadata:
+          labels:
+            label1: value1
+        securityContext:
+          runAsUser: 1000001
+          fsGroup: 0
+        terminationGracePeriodSeconds: 120
+    readinessProbe: <7>
+      initialDelaySeconds: 15
+      timeoutSeconds: 5
+    livenessProbe: <8>
+      initialDelaySeconds: 15
+      timeoutSeconds: 5
+# ...
+----
+<1> Number of Cruise Control application instances; set to `1` to run or `0` to pause the application.
+<2> Specifies the Cruise Control capacity configuration. For more information, see xref:capacity_configuration[capacity configuration].
+<3> Specifies the Cruise Control configuration. You can provide any xref:ref-cruise-control-configuration-{context}[standard configuration option] apart from those managed directly by {ProductName}.
+<4> CPU and memory resources reserved for Cruise Control. For more information, see xref:assembly-resource-limits-and-requests-deployment-configuration-kafka[].
+<5> Logging configuration, to log messages with a given severity (debug, info, warn, error, fatal) or above.
```

Comment:
```suggestion
<5>Defines the logging configuration. Messages are logged with a severity level of `debug`, `info`, `warn`, `error`, or `fatal`.
```

Reply:
I incorrectly  defined the logging configuration here, I have just updated the docs with the correct configuration. Take a look and let me know what you think!

Reply:
New version looks good.